### PR TITLE
Add more room between Oops and Save game

### PR DESCRIPTION
**Pull request by ivangreene**

### CODE REVIEW

File: src/scoreboard/components/Scoreboard/Scoreboard.component.tsx [hunk 1]
```diff
@@ -187,6 +187,8 @@ export const Scoreboard: FC<ScoreboardProps> = () => {
             <input type="button" onClick={toggleEditMode} value={editModeScores ? 'Save changes' : 'Oops'} />
             {editModeScores && <input type="button" onClick={() => setEditModeScores(undefined)} value="Cancel" />}
           </div>
+          <br />
+          <br />
```
Comment (adamldoyle) [inline src/scoreboard/components/Scoreboard/Scoreboard.component.tsx]: You can just increase the marginTop from the div below.

**Comment by jameswylyreasint** [comment]
Should add a confirmation dialog on save game instead

File: src/scoreboard/components/Scoreboard/Scoreboard.component.tsx (hunk 1 above)
Comment (ivangreene) [reply src/scoreboard/components/Scoreboard/Scoreboard.component.tsx]: Nice, I used that and also decided that an `hr` would look nice in there as a logical separator

**Comment by ivangreene** [comment]
@jameswylyreasint I added the confirm dialog

**Comment by adamldoyle** [comment]
At some point I'm going to get rid of the save button in favor of auto-saving throughout the game, but this will fix things until that point.

**Pull request merged**